crates/interpreter/src/interpreter/ext_bytecode.rs

```diff
@@ -92,6 +92,18 @@ impl Jumps for ExtBytecode {
                 .offset_from(self.base.bytecode().as_ptr()) as usize
         }
     }
+    #[inline]
+    fn trace_pc(&self) -> usize {
+        match &self.base {
+            Bytecode::Eof(_) => unsafe {
+                // SAFETY: `instruction_pointer` should be at an offset from the start of the bytecode.
+                // In practice this is always true unless a caller modifies the `instruction_pointer` field manually.
+                self.instruction_pointer
+                    .offset_from(self.base.eof().unwrap().raw.as_ptr()) as usize
```

Comment:
I have already made a change so current `pc` is doing the same thing as `trace_pc`

https://github.com/bluealloy/revm/blob/be92e1db21f1c47b34c5a58cfbf019f6b97d7e4b/crates/interpreter/src/interpreter/ext_bytecode.rs#L87-L95

and bytecode() gives self.base.eof().raw here:
https://github.com/bluealloy/revm/blob/be92e1db21f1c47b34c5a58cfbf019f6b97d7e4b/crates/bytecode/src/bytecode.rs#L137-L143